backend/user_management/models.py

```
@@ -26,13 +44,27 @@ class User(Base):
+    brands = relationship("Brand", back_populates="user")
+
+    # Add this relationship for Order
+    orders = relationship("Order", back_populates="user")
+>>>>>>> backend
 
 
 class Brand(Base):
     __tablename__ = "brand"
 
     brand_id = Column(Integer, primary_key=True, index=True, autoincrement=True)
+<<<<<<< HEAD
```

Ruff (0.8.2)

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Unexpected indentation

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Expected a statement

---

SyntaxError: Unexpected indentation

---

Trailing comma missing

(COM812)